.travis.yml

```diff
@@ -2,7 +2,6 @@ sudo: false
 language: python
 
 python:
-  - "2.7"
```

Comment:
I'm not opposed to dropping python 2.7, but this should be in another PR, that also adds [python_requires](https://packaging.python.org/guides/distributing-packages-using-setuptools/#python-requires) in `setup.py`.